packages/core/src/Modifier.ts

```diff
@@ -22,9 +28,17 @@ export class Modifier {
     applyTo(node: VNode): void {
         node.modifiers.prepend(this);
     }
+    /**
+     * Compare two modifiers.
+     *
+     * Meant to be overridden if necessary.
```

Comment:
"Return true if this modifier is the same as the given modifier, false otherwise.

This can be overridden to change the meaning of equality between modifiers."